### PR TITLE
updates for file handler docs

### PR DESCRIPTION
**Pull request by patrick-rodgers**
Updates to the documentation for file handlers with updated registration instructions, index wording changes, and updates on resetting the cache.

### CODE REVIEW

File: docs/file-handlers/index.md [hunk 1]
```diff
@@ -137,8 +137,11 @@ The activation parameters will be lost if not persisted before this redirect occ
 
 You can see an example of using a data model object and handler method for caching the activation parameters in a cookie, in the [Markdown File Handler sample][markdown-file-handler].
 
+### File Handler example solutions
+
 [GPXFileHandler]: https://github.com/OfficeDev/GPX-FileHandler
 [markdown-file-handler]: https://github.com/rgregg/o365-markdown-file-handler
+[markdown-file-handler]: https://github.com/rgregg/o365-markdown-file-handler
```
[the remark this answers is not in the document]
Comment (chackman) [reply docs/file-handlers/index.md:144]: Maybe the issue is that the header and "You can see an example of using a data model object" blurb are transposed.

**Review by chackman — CHANGES_REQUESTED** [review]
Thanks! The example appears incorrect in the rich preview - was the header transposed with the example, and/or the links supposed to be visible?

File: docs/file-handlers/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrick-rodgers) [reply docs/file-handlers/index.md]: Thanks @chackman - I totally missed coming back to fix up those links while I was editing. Have fixed them and reformatted, looks good now to me in preview.

[the remark this answers is not in the document]
Comment (chackman) [reply docs/file-handlers/index.md]: Thanks!

**Review by chackman — APPROVED** [review]

**Review by chackman — CHANGES_REQUESTED** [review]
Patrick - the robot caught an issue with the change.

The [markdown-file-handler] link reference in the doc needs a valid URL associated with it.
I'll add more comments to the place(s) to fix up.

File: docs/file-handlers/index.md [hunk 2]
```diff
@@ -137,8 +137,11 @@ The activation parameters will be lost if not persisted before this redirect occ
 
 You can see an example of using a data model object and handler method for caching the activation parameters in a cookie, in the [Markdown File Handler sample][markdown-file-handler].
```
Comment (chackman) [inline docs/file-handlers/index.md:138]: The `[markdown-file-handler]` reference needs a link associated with it.
There is no `[markdown-file-handler]` link defined later in the file.  While you have links defined for `[markdown-file-handler (C#)]` and `[markdown-file-handler (TypeScript)]` those are not the same name as `[markdown-file-handler]`. Could you please modify the `[markdown-file-handler]` reference name to point to either the C# or TypeScript examples?

**Review by chackman — CHANGES_REQUESTED** [review]

File: docs/file-handlers/index.md (hunk 2 above)
Comment (patrick-rodgers) [reply docs/file-handlers/index.md]: Got it, thanks. I will watch the build and see if things pass.

**Review by chackman — APPROVED** [review]

**Pull request merged**